client/embedded-components/index.tsx

```diff
@@ -142,13 +137,11 @@ export const EmbeddedAccountOnboarding: React.FC< EmbeddedAccountOnboardingProps
 	onLoaderStart,
 	onLoadError,
 	onStepChange,
-	isPoEligible = false,
```

Comment:
Please can you also remove the `isPoEligible` from the docblock for this component?

Reply:
Done in https://github.com/Automattic/woocommerce-payments/pull/10718/commits/03d304d2e773c704e3ed011ac7af1300aa4dff52, thanks!